packages/fx-core/tests/plugins/solution/solution.provision.test.ts

```diff
@@ -253,6 +253,19 @@ class MockedAzureTokenProvider implements AzureAccountProvider {
   async setSubscription(subscriptionId: string): Promise<void> {
     return;
   }
+  getAccountInfo(): Record<string, string> | undefined {
+    return {};
+  }
+  getSubscriptionId(): string | undefined {
+    return "12";
```

Comment:
Why "12"...

Reply:
Just a valid string, only used for mock test.